sygnal.yaml.sample

```diff
@@ -199,30 +199,24 @@ metrics:
 # For the type, you may specify a fully-qualified Python classname if desired.
 #
 apps:
-  # This is an example APNs push configuration using certificate authentication.
+  # This is an example APNs push configuration
   #
-  #com.example.myapp.ios:
+  #com.example.myapp2.ios:
```

Comment:
Any particular reason for the `myapp2` here? Seems confusing if there's no `myapp`?

Reply:
Oops, simply because there used to be a `myapp` and `myapp2` before the deduplication.